.github/workflows/ci.yml

```
@@ -0,0 +1,44 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+
+jobs:
+  test:
+    name: Test PHP ${{ matrix.php-version }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version: ['7.3', '7.4', '8.0', '8.1', '8.2', '8.3']
```

Looking at https://www.php.net/supported-versions I think we can be a bit more restrictive on this, to save CI minutes. Only 8.3 + 8.4 are in active support, and 8.1 security fixes is ending soon

```suggestion
        php-version: ['8.2', '8.3', '8.4']
```